chromium_src/chrome/browser/media/router/discovery/dial/dial_service_impl.cc

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/media/router/discovery/dial/dial_service_impl.h"
+
+#include "components/version_info/version_info.h"
+
+namespace version_info {
+constexpr std::string_view GetProductNameForChrome() {
+  return "Google Chrome";
+}
+}  // namespace version_info
+
+#define GetProductName GetProductNameForChrome
```

Comment:
this only changes the product name. The user agent will look pretty weird for the receiving side.

Reply:
What do you mean? The User Agent with this change looks like: `USER-AGENT: Google Chrome/122.1.64.0 Mac OS X\r\n` on Mac. Do you mean the version string is out of sync?

Reply:
`Google Chrome/122.1.64.0`

`Google Chrome` version has different values in minor/build/patch fields, i.e. minor is always 0, build is always in thousands. Not sure how critical this is in terms of compatibility, that's why I added @fmarier to have a second look here.

Reply:
I changed it so as to match the Version Number too.

Reply:
@ShivanKaul says the new version number looks like this now:
```
Google Chrome/122.0.6261.29 Mac OS X\r\n
```